tests/queries/0_stateless/01872_initial_query_start_time.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -ue
+
+# this test doesn't need 'current_database = currentDatabase()',
+
+unset CLICKHOUSE_LOG_COMMENT
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+${CLICKHOUSE_CLIENT} -q "drop table if exists m"
+${CLICKHOUSE_CLIENT} -q "create table m (dummy UInt8) ENGINE = Distributed('test_cluster_two_shards', 'system', 'one')"
+
+query_id=$(${CLICKHOUSE_CLIENT} -q "select lower(hex(reverse(reinterpretAsString(generateUUIDv4()))))")
+${CLICKHOUSE_CLIENT} -q "select * from m format Null" "--query_id=$query_id"
+
+${CLICKHOUSE_CLIENT} -n -q "
+system flush logs;
+select
+    anyIf(initial_query_start_time, is_initial_query) = anyIf(initial_query_start_time, not is_initial_query),
+    anyIf(initial_query_start_time_microseconds, is_initial_query) = anyIf(initial_query_start_time_microseconds, not is_initial_query)
+from system.query_log
+where initial_query_id = '$query_id' and type = 'QueryFinish';
```

Comment:
Just in case

```suggestion
where event_date = today() and event_time >= now() - interval 1 hour and initial_query_id = '$query_id' and type = 'QueryFinish';
```

Reply:
ok.